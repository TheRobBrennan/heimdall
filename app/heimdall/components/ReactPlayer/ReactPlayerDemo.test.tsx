import React from "react"
import TestRenderer from "react-test-renderer"

import Component from "./ReactPlayerDemo"

describe("The ReactPlayerDemo component", () => {
  it(`should render`, () => {
    // Verify success state
    const component = TestRenderer.create(<Component />)
    expect(component.toJSON()).toMatchSnapshot()
  })
})
