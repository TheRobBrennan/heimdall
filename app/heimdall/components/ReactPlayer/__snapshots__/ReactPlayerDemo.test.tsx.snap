// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The ReactPlayerDemo component should render 1`] = `
<div
  style={
    Object {
      "height": "360px",
      "margin": "auto",
      "width": "90%",
    }
  }
/>
`;
